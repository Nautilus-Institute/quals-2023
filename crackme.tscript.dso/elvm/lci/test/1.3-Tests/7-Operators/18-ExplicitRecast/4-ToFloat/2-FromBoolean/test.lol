HAI 1.3
	I HAS A var1 ITZ FAIL
	I HAS A var2 ITZ WIN
	var1 IS NOW A NUMBAR
	var2 IS NOW A NUMBAR
	VISIBLE var1
	VISIBLE var2
KTHXBYE
