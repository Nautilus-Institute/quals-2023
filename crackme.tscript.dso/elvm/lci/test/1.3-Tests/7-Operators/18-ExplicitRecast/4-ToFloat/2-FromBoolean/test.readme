This test checks that the explicit recast operator correctly casts boolean
values to floating point decimal types.
