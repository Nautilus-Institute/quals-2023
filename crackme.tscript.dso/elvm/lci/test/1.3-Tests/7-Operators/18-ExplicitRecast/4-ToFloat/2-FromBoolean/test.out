0.00
1.00
